.github/workflows/coverage-test.yml

```diff
@@ -0,0 +1,42 @@
+name: CI - Test Coverage
+
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch: {}
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
```

Comment:
Sorry, we still have plenty of comments explaining how GitHub action works. Can you remove them?